Order.Management/Program.cs

```diff
@@ -8,117 +11,105 @@ class Program
         // Main entry
         static void Main(string[] args)
         {
-            var (customerName, address, dueDate) = CustomerInfoInput();
+            var customerInfo = CustomerInfoInput();
```

Comment:
Using Tuple is a good practice but here I think having a CustomerInfo class is good for the long term.